app/models/repository.rb

```diff
@@ -107,8 +109,8 @@ class Repository < ApplicationRecord
 
   before_save :normalize_license_and_language
   after_commit :update_all_info_async, on: :create
-  after_commit :save_projects, on: :update
-  after_commit :update_source_rank_async, on: [:update]
+  after_commit :update_source_rank_async, on: [:update], if: -> { previous_changes.except("updated_at").any? }
+  after_commit :send_repository_updated, on: %i[create update]
```

Comment:
should we limit this to just changes on the fields defined in the new serializer?

Reply:
there's no new serializer proposed for repo though, only project?

(though I have been thinking we might want the repo updated event to have more fields in that same spirit)

Reply:
ah I see. we only really are sending the name of the repo and updated_at, and then the expectation is taht someone will call an API later to get the actual new data (like maint stats)

Reply:
got it! Makes sense then